src/Symfony/Bridge/Twig/composer.json

```diff
@@ -30,7 +30,7 @@
         "symfony/http-kernel": "~3.4|~4.0",
         "symfony/mime": "~4.3",
         "symfony/polyfill-intl-icu": "~1.0",
-        "symfony/routing": "~3.4|~4.0",
+        "symfony/routing": "~4.3",
```

Comment:
This doesn't look needed anymore. Maybe the constraints for the HttpFoundation component need to be changed instead?